projects/cdk/directives/for-async/for-async.directive.ts

```diff
@@ -0,0 +1,59 @@
+import {
+    Directive,
+    Inject,
+    Input,
+    NgZone,
+    OnChanges,
+    OnDestroy,
+    TemplateRef,
+    ViewContainerRef,
+} from '@angular/core';
+import {tuiZonefree} from '@taiga-ui/cdk/observables';
+import {from, of, Subject} from 'rxjs';
+import {concatMap, delay, takeUntil} from 'rxjs/operators';
+
+@Directive({selector: '[tuiForAsync][tuiForAsyncOf]'})
+export class TuiForAsyncDirective<T> implements OnChanges, OnDestroy {
+    private readonly destroy$ = new Subject<void>();
```

Comment:
Why not `TuiDestroyService`?

Reply:
<img width="835" alt="image" src="https://user-images.githubusercontent.com/12021443/170513408-5442e4ad-d8db-4c69-aaa9-6fee8ed77f5b.png">

Because when I use it, if I call the next method in order to prevent the race condition, then everything stops working.